test/integration/webhook/suite_test.go

```diff
@@ -91,12 +93,17 @@ var _ = BeforeSuite(func() {
 	// start webhook server using Manager
 	webhookInstallOptions := &testEnv.WebhookInstallOptions
 	mgr, err := ctrl.NewManager(cfg, ctrl.Options{
-		Scheme:             scheme.Scheme,
-		Host:               webhookInstallOptions.LocalServingHost,
-		Port:               webhookInstallOptions.LocalServingPort,
-		CertDir:            webhookInstallOptions.LocalServingCertDir,
-		LeaderElection:     false,
-		MetricsBindAddress: "0",
+		Scheme:         scheme.Scheme,
+		LeaderElection: false,
+		Metrics: server.Options{
+			BindAddress: "0",
```

Comment:
I see setting this to `0` will disable the metrics server. This seems like it would prevent a prometheus operator from scraping metrics as described [here](https://github.com/kubernetes-sigs/jobset/blob/main/docs/setup/install.md#optional-add-metrics-scraping-for-prometheus-operator) (please correct me if I'm wrong)?

Reply:
This is testing for the webhooks. And we had it set to 0 before.

Reply:
Ah I see, ok thanks